### ISSUE TITLE
Toggle read only icon

### ISSUE THREAD

**Issue by redforkian** [giuspen/cherrytree#2458]
If a node on the left side has a long title, the lock icon showing when it is read only sometimes will not show until you resize the window.
Perhaps show the read only icon on the left side of title instead?

**Comment by firepainting** [comment]
I agree. The best place for the lock icon would be on the left side (in between cherries and node names). I also enter long node names and I thought something was wrong with my CTB file. (I have another file with shorter node names.)

This is the lowest width for the tree explorer where the lock icons begin to appear on my main file:

<img width="1025" height="576" alt="Image" src="https://github.com/user-attachments/assets/a02f9a33-6048-458f-93d2-3bb749914b97" />

**Comment by giuspen** [comment]
If you are happy to test a pre-release including this please find it at https://www.giuspen.net/cherrytree/#testing (called v1.7.1+8)

**Comment by firepainting** [comment]
Hi @giuspen, since yesterday on my end, your site https://www.giuspen.net has been "taking too long to respond," it wouldn't load. (I was gonna check something on the manual.) Anyway, taking this opportunity to tell you that this has also happened occassionally in the past. I would say maybe around 15% of the time I'd visit the site, but I only do that sometimes. It has happened "quite a lot" for a low visit frequency for me to remember.

Anyway, I would love to test that pre-release of course! (Once I get access.) Many thanks as usual.

**Comment by giuspen** [comment]
Hi @firepainting please try downloading from https://github.com/giuspen/cherrytree/actions/runs/32054649339 although that will become invalid in a day, in case you tell me I will run another build and give you the new link (scroll down, download the artifact, extract the zip and find the installer and portable inside)

**Comment by firepainting** [comment]
Hi @giuspen. I was away for a bit. I just downloaded the artifact but it looks like it's only for Windows? I'm on Linux Mint 21.3. Thanks!

**Comment by giuspen** [comment]
Ho @firepainting please look at https://github.com/giuspen/cherrytree/actions/runs/32295757353 for the Linux AppImage

**Comment by firepainting** [comment]
Looks fine on my end! Thank you @giuspen! 👍️

<img width="1161" height="653" alt="Image" src="https://github.com/user-attachments/assets/88d67cb8-e040-4aab-a8ca-c22a6841f414" />
